### PR TITLE
feat: adds a config for documentation button

### PR DESCRIPTION
**Pull request by faouziH21**
depends on: https://github.com/terrestris/shogun-docker/pull/163

### CODE REVIEW

File: src/components/UserMenu/index.tsx [hunk 1]
```diff
@@ -193,19 +193,16 @@ export const UserMenu: React.FC<UserProps> = (): JSX.Element => {
...
-        divider,
-        logout
+        info
       ];
-
+      if (ClientConfiguration.documentationButtonVisible) {
+        itemsForLoggedInUser.push(docs);
+      }
+      itemsForLoggedInUser.push(divider, logout);
       items.push(...itemsForLoggedInUser);
```
[the remark this answers is not in the document]
Comment (LukasLohoff) [reply src/components/UserMenu/index.tsx:206]: should the documentation button be visible as well? if that is the case, I think you'll have to also make adjustments in line 185

[the remark this answers is not in the document]
Comment (faouziH21) [reply src/components/UserMenu/index.tsx:206]: PM says it's ok for now.

**Review by LukasLohoff — APPROVED** [review]

File: src/global.d.ts [hunk 2]
```diff
@@ -60,6 +60,7 @@ declare module 'clientConfig' {
     search?: SearchConfiguration;
     featureEditRoles?: FeatureEditConfiguration;
     wfsLockFeatureEnabled?: boolean;
+    documentationButtonVisible?: boolean;
```
Comment (dnlkoch) [inline src/global.d.ts:63]: Could you please add the newly added parameter to the docs in the [README](https://github.com/terrestris/shogun-gis-client?tab=readme-ov-file#configuration-)?

Comment (faouziH21) [reply src/global.d.ts:63]: see: https://github.com/terrestris/shogun-gis-client/pull/1740

**Pull request merged**